docs/app/pages/components/components-sections/input-controls/number-picker/number-picker.component.html

```diff
@@ -48,8 +48,8 @@
         Can be used to show a red outline around the input to indicate an invalid value. By default the
         error state will appear if the user enters a number below the minimum value or above the maximum value.
     </tr>
-    <tr uxd-api-property name="precision" type="number" defaultValue="6">
-        Define the precision of floating point values.
+    <tr uxd-api-property name="precision" type="number">
```

Comment:
Maybe adding `defaultValue="15"` here would be less confusing than the current description.